### PR TITLE
[Keyboard] Update ZSA Keyboard Readmes and keymaps

### PR DESCRIPTION
**Pull request by drashna**
## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/moonlander/readme.md [hunk 1]
```diff
@@ -19,7 +19,23 @@ Flashing example for this keyboard:
...
+There are 6 functions for enabling and disabling the LEDs on the top of the boards. The functions are `ML_LED_1(bool)` through `ML_LED_6(bool)`, with the first LED being the left most LED on the left hand, and the sixth LED being the right most LED on the right side. 
+
+By default, the Indicator LEDs are used to indicate the layer state for the keyboard.  If you wish to change this (and indicate caps/num/scroll lock status instead), then define `MOONLANDER_USER_LEDS` in your `config.h` file. 
+
+
+### Oryx Configuation
+
+To enable the features from Oryx (ZSA's Configurator), either compile the the `default` keymap, or add `#define ORYX_CONFIGURATOR` to your `config.h` file.  
+
+This enableds the front Indicator LEDs, and the `TOGGLE_LAYER_COLOR` keycode.  The `TOGGLE_LAYER_COLOR` keycode toggles the customized LED map configured on Oryx.
```
Comment (fauxpark) [inline keyboards/moonlander/readme.md:36]:
```suggestion
This enables the front Indicator LEDs, and the `TOGGLE_LAYER_COLOR` keycode.  The `TOGGLE_LAYER_COLOR` keycode toggles the customized LED map configured on Oryx. 
```

File: keyboards/ergodox_ez/rules.mk [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
 
-# Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
+
```
Comment (fauxpark) [inline keyboards/ergodox_ez/rules.mk:4]:
```suggestion
# Bootloader selection
```

File: keyboards/ergodox_ez/keymaps/default/keymap.c [hunk 3]
```diff
@@ -115,94 +116,95 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+    if (record->event.pressed) {
+        switch (keycode) {
+            case VRSN:
+                SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+                return false;
+#ifdef RGBLIGHT_ENABLE
+            case RGB_SLD:
+                rgblight_mode(1);
+                return false;
+#endif
```
Comment (fauxpark) [inline keyboards/ergodox_ez/keymaps/default/keymap.c:131]: Couldn't this be replaced with `RGB_M_P`?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ergodox_ez/keymaps/default/keymap.c (hunk 3 above)
Comment (drashna) [reply keyboards/ergodox_ez/keymaps/default/keymap.c]: Yup, and done

File: keyboards/moonlander/readme.md [hunk 4]
```diff
@@ -19,7 +19,23 @@ Flashing example for this keyboard:
...
+## Moonlander Customization
+
+### Indicator LEDs
+
+There are 6 functions for enabling and disabling the LEDs on the top of the boards. The functions are `ML_LED_1(bool)` through `ML_LED_6(bool)`, with the first LED being the left most LED on the left hand, and the sixth LED being the right most LED on the right side. 
+
+By default, the Indicator LEDs are used to indicate the layer state for the keyboard.  If you wish to change this (and indicate caps/num/scroll lock status instead), then define `MOONLANDER_USER_LEDS` in your `config.h` file. 
+
+
+### Oryx Configuation
```
Comment (fauxpark) [inline keyboards/moonlander/readme.md:32]:
```suggestion
### Oryx Configuration
```

**Review by fauxpark — APPROVED** [review]

File: keyboards/planck/ez/readme.md [hunk 5]
```diff
@@ -4,24 +4,26 @@
...
-    make planck/ez/glow:default
+    make planck/ez/glow:oryx
+    
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
 
 ## Planck EZ Configuration (from Oryx)
 
+To enable the features from Oryx (ZSA's Configurator), either compile the the `oryx` keymap, or add `#define ORYX_CONFIGURATOR` to your `config.h` file.  
+
+This enableds the front Indicator LEDs, and the `TOGGLE_LAYER_COLOR`, and `LED_LEVEL` keycodes.  The `TOGGLE_LAYER_COLOR` keycode toggles the customized LED map configured on Oryx. The `LED_LEVEL` cycles through the brightness levels for the front "teeth" LEDs.
```
Comment (fauxpark) [inline keyboards/planck/ez/readme.md:25]:
```suggestion
This enables the front Indicator LEDs, and the `TOGGLE_LAYER_COLOR`, and `LED_LEVEL` keycodes.  The `TOGGLE_LAYER_COLOR` keycode toggles the customized LED map configured on Oryx. The `LED_LEVEL` cycles through the brightness levels for the front "teeth" LEDs. 
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Last one, I promise 😅

**Review by fauxpark — APPROVED** [review]

**Pull request merged**